snappy/rollback.go

```diff
@@ -31,25 +30,5 @@ import (
 //
 // The version needs to be installed on disk
 func Rollback(pkg, ver string, inter progress.Meter) (version string, err error) {
```

Comment:
This really concerns me. I know this is mean to be temporary, but it is still making me really nervous. The rollback story is the biggest feature compared to traditional systems. Combined with the boot logic to auto rollback for broken boots its a very strong story. Breaking that even temporary does not feel right. Sorry for this negative review.